lang/cs/Org.Apache.REEF.IMRU/OnREEF/IMRUTasks/UpdateTaskHost.cs

```diff
@@ -43,23 +42,28 @@ public sealed class UpdateTaskHost<TMapInput, TMapOutput, TResult> : ITask
         private readonly IBroadcastSender<MapInputWithControlMessage<TMapInput>> _dataAndControlMessageSender;
         private readonly IUpdateFunction<TMapInput, TMapOutput, TResult> _updateTask;
         private readonly bool _invokeGC;
+        private readonly IObserver<TResult> _resultHandler;
```

Comment:
I had it earlier. @markusweimer  made me change it. However, I agree with you. Let me change it.